### PR TITLE
SIMD-0196: Migrate Stake to Core BPF

### PR DESCRIPTION
**Pull request by 2501babe**

### CODE REVIEW

**Review by buffalojoec — APPROVED** [review]
Lgtm

**Comment by Benhawkins18** [comment]
Would like to get an approval/ buy in from somebody on the FD team before merging.

**Review by joncinque — APPROVED** [review]

**Review by topointon-jump — APPROVED** [review]

**Comment by joncinque** [comment]
It looks like there's consensus here -- if there are no comments within the next week, I will merge this on December 4.

**Review by ripatel-fd — APPROVED** [review]

**Pull request merged**
